VERSION

```diff
@@ -0,0 +1 @@
+1.0.0-pre1
```

Comment:
The documentation for the tile contents seems like it belongs in this repo, along with the code which creates those contents... Is it the URL structure and available formats which is determined externally? In which case, perhaps it's worth splitting the docs about those different things?